db/schema.rb

```diff
@@ -312,7 +312,7 @@
     t.datetime "created_at", null: false, comment: "sensitive=false"
     t.datetime "updated_at", null: false, comment: "sensitive=false"
     t.boolean "current_address_matches_id", comment: "True if the user indicates that their current address matches the address on the ID they're bringing to the Post Office. sensitive=false"
-    t.jsonb "selected_location_details", comment: "The location details of the Post Office the user selected (including title, address, hours of operation) sensitive=false"
+    t.jsonb "selected_location_details", comment: "sensitive=true"
```

Comment:
we should append to this comment instead of overwriting it

Reply:
Apologies for the oversight. I'll correct it. Thanks.